### PR TITLE
ci: fix v8 cppgc files in wrong location

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Refs [2139585](https://chromium-review.googlesource.com/c/v8/v8/+/2139585).

V8 moved a few files used by Node.js to another directory in the most recent Chromium roll, which clashed with the changes made in the latest Node.js roll. This fixes those incorrect locations.

cc @nornagon @MarshallOfSound @jkleinsc 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: patches/node/build_add_gn_build_files.patch [hunk 1]
```diff
@@ -944,7 +944,25 @@ index 0000000000000000000000000000000000000000..78dff9e0458aaa6b864259f3001764c7
...
++        "//v8/include/cppgc/source-location.h",
++        "//v8/include/cppgc/trace-trait.h",
++        "//v8/include/cppgc/type-traits.h",
++        "//v8/include/cppgc/visitor.h"
++      ]
++    },
++    {
++      "dest_dir": "include/node//cppgc/internal/",
++      "files": [
++        "//v8/include/cppgc/internal/accessors.h",
```
Comment (deepak1556) [inline patches/node/build_add_gn_build_files.patch:961]: Why are we exposing internal headers through `node_headers` ?

Comment (deepak1556) [reply patches/node/build_add_gn_build_files.patch:961]: The CL commit says,

```
cppgc: Move implementation headers in internal/ dir

Headers containing only implementation details are better to stay in a
separate dir to indicate the user that they shouldn't be included.
```

Comment (codebytere) [reply patches/node/build_add_gn_build_files.patch:961]: @deepak1556 hmm - i just ran `python tools/generate_gn_filenames_json.py` from the node dir

Comment (nornagon) [reply patches/node/build_add_gn_build_files.patch:961]: @deepak1556 the non-internal headers include the internal ones. it just means "shouldn't be included _directly_".

Comment (deepak1556) [reply patches/node/build_add_gn_build_files.patch:961]: Okay in that case there is nothing we can do, I was hoping the non-internal ones are like `v8.h`

**Pull request merged**